### PR TITLE
Fix issue #1037: Task inbox small design improvements

### PR DESCRIPTION
**Pull request by kelvinyelyen**
This pull request includes the following changes:

1. [Already Fixed] H3 Font weight is 700 instead of 600
2. [Already Fixed] Task title is black instead of Text primary
3. Additionally, the tab switcher behavior has been updated to match Figma specifications: Added Hover effect.

Please review the adjustments and let me know if any further changes or refinements are necessary.

### CODE REVIEW

File: front/src/modules/ui/tab/components/Tab.tsx [hunk 1]
```diff
@@ -21,12 +21,19 @@ const StyledTab = styled.div<{ active?: boolean }>`
   display: flex;
   gap: ${({ theme }) => theme.spacing(1)};
   justify-content: center;
-  padding: ${({ theme }) => theme.spacing(2) + ' ' + theme.spacing(4)};
+  padding: ${({ theme }) => theme.spacing(2) + ' ' + theme.spacing(2)};
+`;
+
+const HoverSpan = styled.span`
```
Comment (Weiko) [inline front/src/modules/ui/tab/components/Tab.tsx:27]: We tend to use the prefix 'Styled' to distinguish them from actual components.
See StyledTab above.

**Review by Weiko — COMMENTED** [review]
Left a comment otherwise LGTM, thank you @kelvinyelyen!

File: front/src/modules/ui/tab/components/Tab.tsx (hunk 1 above)
Comment (kelvinyelyen) [reply front/src/modules/ui/tab/components/Tab.tsx:27]: Thanks for your feed back! Updating...

Comment (kelvinyelyen) [reply front/src/modules/ui/tab/components/Tab.tsx]: I've made the updates by adding the 'Styled' prefix to the styled components, as you suggested. This should help improve clarity and consistency in our codebase. Please let me know if the changes meet your expectations.

Comment (Weiko) [reply front/src/modules/ui/tab/components/Tab.tsx]: @kelvinyelyen thanks!

**Review by Weiko — APPROVED** [review]

**Pull request merged**